pkg/lang/ir/parser/conda.go

```diff
@@ -24,6 +24,7 @@ type CondaEnv struct {
 	EnvName       string
 	CondaPackages []string
 	PipPackages   []string
+	Channels      []string
```

Comment:
Could you please add a e2e test case in e2e/lang?

Reply:
done